uint32 number_of_drones # used to be uint8 stupid variable
float64 reach_distance
uint32 solver_type
float64 obs_check_distance
float64 rescaling_factor
float64 intended_velocity



